### PR TITLE
RUL-38: Add copiers for collection attributes

### PR DESCRIPTION
**Pull request by mmetayer**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/CollectionToStringDataConverter.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public function convert(ValueInterface $sourceValue, AttributeInterface $targetAttribute)
+    {
+        Assert::isIterable($sourceValue->getData());
+
+        $chunks = [];
+        foreach ($sourceValue->getData() as $data) {
+            $chunks[] = (string) $data;
+        }
+
+        return implode(', ', $chunks);
```
Comment (nmarniesse) [inline src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/CollectionToStringDataConverter.php]: So we use the code of the options.ref entities? Not the label?

File: src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/StringToStringDataConverter.php [hunk 2]
```diff
@@ -26,6 +26,16 @@ final class StringToStringDataConverter implements ValueDataConverter
...
+        AttributeTypes::OPTION_SIMPLE_SELECT => [
+            AttributeTypes::TEXT => true,
+            AttributeTypes::TEXTAREA => true,
+            AttributeTypes::REFERENCE_ENTITY_SIMPLE_SELECT => true,
+        ],
+        AttributeTypes::REFERENCE_ENTITY_SIMPLE_SELECT => [
+            AttributeTypes::TEXT => true,
+            AttributeTypes::TEXTAREA => true,
+            AttributeTypes::OPTION_SIMPLE_SELECT => true,
+        ],
```
Comment (nmarniesse) [inline src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/StringToStringDataConverter.php]: Same question: only the code is copied, not the label?

File: src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/CollectionToStringDataConverter.php (hunk 1 above)
Comment (mmetayer) [reply src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/CollectionToStringDataConverter.php]: Nope, we use the codes

File: src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/StringToStringDataConverter.php (hunk 2 above)
Comment (mmetayer) [reply src/Akeneo/Pim/Enrichment/Component/Product/Updater/Converter/StringToStringDataConverter.php]: same answer :)

**Pull request merged**